crypto/evp/cmeth_lib.c

```diff
@@ -188,3 +190,4 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
     return cipher->ctrl;
 }
 
+#endif  /* OPENSSL_API_3 */
```

Comment:
Remove this line - the #ifdef was removed when you moved the exclusion into build.info